fmriprep/workflows/fieldmap/unwarp.py

```diff
@@ -447,7 +447,8 @@ def init_prepare_epi_wf(ants_nthreads, name="prepare_epi_wf"):
     workflow = pe.Workflow(name=name)
 
     def _flatten(l):
-        return [item for sublist in l for item in sublist]
+        from niworkflows.nipype.utils.filemanip import filename_to_list
+        return [item for sublist in l for item in filename_to_list(sublist)]
```

Comment:
Guess I'll just request a review. So the situation is that the input to `merge` from `split` is being interpreted as a list of single-character strings.

Given that the output is passed to `_flatten` before being passed to `merge`, I believe the correct interpretation is that we get a list with a single element `['/path/to/file']`. Expanding that out:

```Python
[item for sublist in ['/path/to/file'] for item in sublist]
[item for item in '/path/to/file']
['/', 'p', 'a', 't', 'h', '/', 't', 'o', '/', 'f', 'i', 'l', 'e']
```

Thus, by using `filename_to_list` in the inner loop, we get:

```Python
[item for sublist in ['/path/to/file'] for item in filename_to_list(sublist)]
[item for item in filename_to_list('/path/to/file')]
[item for item in ['/path/to/file']]
['/path/to/file']
```

I believe that because this is the output of a `MapNode`, the original input must always be a list, so there's no concern about `l` itself being a string. However, we could be more cautious for basically no cost and apply `filename_to_list` at both the inner and outer loops.